FrontEnd/src/components/adminPage/detail-view/ProfileDetail.jsx

```diff
@@ -0,0 +1,129 @@
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router';
+import DeleteModal from './DeleteModal';
+import css from './ProfileDetail.module.css';
+import axios from 'axios';
+
+
+function ProfileDetail() {
+    const [deleteModalActive, setDeleteModalActive] = useState(false);
+    const [error, setError] = useState(null);
+    const [profile, setProfile] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [updateSuccess, setUpdateSuccess] = useState(false);
+    const profileId = usePathCompanyId();
+    const url = `${process.env.REACT_APP_BASE_API_URL}/api/admin/profiles/${profileId}/`;
+    const navigate = useNavigate();
+    const companyInfo = [
+        { label: 'Ім\'я', key: 'name' },
+        { label: 'person_position', key: 'person_position' },
+        { label: 'official_name', key: 'official_name' },
+        { label: 'region', key: 'region' },
+        { label: 'phone', key: 'phone' },
+        { label: 'edrpou', key: 'edrpou' },
+        { label: 'address', key: 'address' },
+    ];
+
+    useEffect(() => {
+
+        const fetchData = async () => {
+            try {
+                const response = await axios.get(url);
+                setProfile(response.data);
+                setLoading(false);
+            } catch (error) {
+                setError(error.message);
+                setLoading(false);
+            }
+        };
+        fetchData();
+
+    }, [url, profileId]);
+
+    const handleSaveChanges = async () => {
+        try {
+            const response = await axios.put(
+                url,
+                {
+                    name: profile.name,
+                    is_deleted: profile.is_deleted,
+                },
+            );
+            if (response.status !== 200) {
+                throw new Error(`HTTP error! Status: ${response.status}`);
+            }
+            setUpdateSuccess(true);
+        } catch (error) {
+            console.error('Failed to update profile:', error);
+        }
+    };
+
+    if (loading) {
+        return <p>Loading...</p>;
+    }
+
+    if (error) {
+        return <p>Error: {error}</p>;
+    }
+
+    const handleDeleteUser = async () => {
+        try {
+            const response = await axios.delete(url);
+            if (response.status !== 204) {
+                throw new Error(`HTTP error! Status: ${response.status}`);
+            }
+            setProfile([]);
+            navigate('/customadmin/profiles');
+        } catch (error) {
+            console.error('Failed to delete profile:', error);
```

Comment:
Do we need to show the toast message to notify the user that smth is wrong?

Reply:
Done